app/page.tsx

```
@@ -69,9 +70,9 @@ function isSharingData(data: unknown): data is { members: ProjectMember[]; invit
 
 function HomeContent() {
   // All hooks at the top
-  const { user } = useUser()
-  const { signOut } = useClerk()
-  const { session, isLoaded } = useSession()
+  const { user } = BYPASS_CLERK ? { user: null } : useUser()
```

ESLint

[error] React Hook "useUser" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)